packages/client/hmi-client/src/workflow/operator/tera-operator-image-carousel.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+	<Galleria v-if="!isEmpty(images)" :value="images" :showThumbnails="false" :showIndicators="true">
+		<template #item="slotProps">
+			<img :src="slotProps.item" alt="Alt" />
+		</template>
+	</Galleria>
+	<div v-else class="no-image">
+		<i class="pi pi-image" />
+	</div>
+</template>
+
+<script setup lang="ts">
+import { isEmpty } from 'lodash';
+import { PropType } from 'vue';
+import Galleria from 'primevue/galleria';
+
+defineProps({
+	images: {
+		type: Array as PropType<string[]>,
+		default: () => []
+	}
+});
```

Comment:
Can you add some comments on what format types are supported here? Are we constrained to use image-hrefs or can we also pass in base64 image data (as strings)? 

e.g. We can display the images, but where are we planning on storing said images and if the images change due to modelers' interactions?